src/main/java/net/minecraftforge/internal/ForgeExceptionFactories.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Minecraft Forge - Forge Development LLC
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.internal;
+
+public class ForgeExceptionFactories {
+
+	public static final IllegalCallerException INTERNAL_METHOD = new IllegalCallerException("This is an internal forge method and cannot be used by mods");
```

Comment:
This whole class for one field seems a bit weird...

Reply:
Also, won't static init here really mess with the displayed stacktrace?